src/main/java/webdriver/DriverFactory.java

```diff
@@ -0,0 +1,32 @@
+package webdriver;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeDriverService;
+
+import java.util.concurrent.TimeUnit;
+
+public class DriverFactory {
+    private static final String CHROMEDRIVER_EXE_PATH = "src/main/resources/chromedriver.exe";
+    public static final int IMPLICITLY_WAIT = 10;
+
+    public enum BrowserType {FIREFOX, CHROME, EDGE, IE}
+
+    public static WebDriver getDriver(BrowserType type) {
+        WebDriver driver = null;
+        switch (type) {
+            case CHROME:
+                driver = getChromeDriver();
+                break;
+        }
+        driver.manage().window().maximize();
```

Comment:
you need to add a default section to the switch above becuase if you pass any other browsertype you will get nullpointer on the 22nd line as your switch will be skipped and driver won't be initialized